OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -695,6 +695,21 @@ internal static void UpgradeActorRules(ModData modData, int engineVersion, ref L
 						RenameNodeKey(node, "DisableOnCondition");
 				}
 
+				// Replaced NukePower CameraActor with CameraRange (effect-based reveal)
+				if (engineVersion < 20161216)
+				{
+					var nukePower = node.Value.Nodes.FirstOrDefault(n => n.Key.StartsWith("NukePower"));
+					if (nukePower != null)
+					{
+						var cameraActor = nukePower.Value.Nodes.FirstOrDefault(n => n.Key == "CameraActor");
+						if (cameraActor != null)
+						{
+							nukePower.Value.Nodes.Remove(cameraActor);
+							nukePower.Value.Nodes.Add(new MiniYamlNode("CameraRange", "10"));
```

Comment:
Can you please log a line here saying that CameraActor has been removed and the range needs to be manually migrated?